name-emu/Cargo.toml

```diff
@@ -6,3 +6,7 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+thiserror = "1.0.48"
+dap = "0.3.1-alpha1"
```

Comment:
Is the alpha branch safe to use or should we stick with something more stable?

Reply:
Unfortunately dap-rs does not yet have a stable version. We will have to live with this and keep it up to date